cmd/virt-launcher/virt-launcher.go

```diff
@@ -468,8 +468,11 @@ func main() {
 	finalShutdownCallback := func(pid int) {
 		err := domainManager.KillVMI(vmi)
 		if err != nil {
-			log.Log.Reason(err).Errorf("Unable to stop qemu with libvirt, falling back to SIGTERM")
-			syscall.Kill(pid, syscall.SIGTERM)
+			log.Log.Reason(err).Errorf("Unable to stop qemu with libvirt")
+			if pid != 0 {
+				log.Log.Warning("Falling back to SIGTERM")
+				syscall.Kill(pid, syscall.SIGTERM)
```

Comment:
Can we log the error here?

Reply:
The error is logged few lines above: 
```go
log.Log.Reason(err).Errorf("Unable to stop qemu with libvirt")
```
Do you think it it needs to be logged twice?

Reply:
I am referring to an error from `syscall.Kill`.

Reply:
Ah, sorry for my confusion :) Initially there was no error checking here so I though you were refering to the error above. Definitely, makes sense to log the kill error. Thanks for pointing that,. Fixed.